tools/lint.mjs

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env node
+
+// Requires Node.js and:
+// * npm install node-html-parser
```

Comment:
Should we create a package.json for this dependency?

Reply:
Suggestions welcome. I know very little about how to Node properly! I think I can add the package.json correctly but further suggestions will help.

Reply:
Done in 2b671f4 but please verify!